src/main.go

```diff
@@ -157,20 +156,25 @@ func init() {
 	// init dir GANTED_LOG_DIR
 	gantedLogDir := getEnv("GANTED_LOG_DIR", "/var/log/ganted")
 	if _, err := os.Stat(gantedLogDir); os.IsNotExist(err) {
-		os.Mkdir(gantedLogDir, 0755)
+		if err := os.Mkdir(gantedLogDir, 0755); err != nil {
+			panic(err)
```

Comment:
`log.Fatalf` should do better here. It prints a formatted message and exits the program.

Reply:
I considered it, but I miss it because I think err will contain enough information. But It would be better that we explicitly print message. I fixed it.